src/main/webapp/WEB-INF/views/investment/investments.jsp

```diff
@@ -0,0 +1,62 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="sec" uri="http://www.springframework.org/security/tags" %>
+
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<%--<sec:authorize access="isAuthenticated()">--%>
+<html>
+<head>
+	<title>All investments</title>
+
+	<link rel="shortcut icon" href="${contextPath}/resources/img/favicon.png" type="image/x-icon">
+	<link href="${contextPath}/resources/css/bootstrap.css" rel="stylesheet">
+	<link href="${contextPath}/resources/css/font-awesome.css" rel="stylesheet">
+	<link href="${contextPath}/resources/css/main.css" rel="stylesheet">
+	<link href="${contextPath}/resources/css/card.css" rel="stylesheet">
+
+	<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
+	<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+</head>
+<body>
+<h1>${pageName}</h1>
+<div class="row masonry" data-columns>
+	<c:forEach items="${investmentList}" var="investment">
+		<div class="item">
+			<div class="thumbnail">
+				<div class="caption centered">
+					<ul>
+						<li>
+							<h4>Project name:
```

Comment:
You should have 4 spaces index at max. This is not readable. You have 4 tabs here